CHANGELOG/CHANGELOG-1.11.md

```diff
@@ -1,5 +1,26 @@
 # Changelog 1.11
 
+## 1.11.2 - 2023-05-29
+
+### Changed
+
+- Update Open vSwitch to 2.17.6. ([#4959](https://github.com/antrea-io/antrea/pull/4959), [@tnqn])
+- Bump up whereabouts to v0.6.1. ([#4988](https://github.com/antrea-io/antrea/pull/4988), [@hjiajing])
+
+### Fixed
+
+- Fix the Antrea Agent crash issue on Windows by running modules that rely on Services after AntreaProxy is ready. ([#4946](https://github.com/antrea-io/antrea/pull/4946), [@tnqn])
+- [Multicast] Clone remoteMembers when creating an updated GroupMemberStatus . ([#4903](https://github.com/antrea-io/antrea/pull/4903), [@ceclinux])
```

Comment:
This should be:
```
- Fix the Antrea Agent crash issue which is caused by a concurrency bug in Multicast feature with encap mode. ([#4903](https://github.com/antrea-io/antrea/pull/4903), [@ceclinux])
```
Please go through release 1.12 log which should include all PRs with refined descriptions.

Reply:
Fixed.